docs/settings.rst

```diff
@@ -699,36 +699,37 @@ Following are example ways to specify your preferred theme::
 free to use them in your themes as well.
 
-=======================   =======================================================
-Setting name              What does it do?
-=======================   =======================================================
-``SITESUBTITLE``          A subtitle to appear in the header.
-``DISQUS_SITENAME``       Pelican can handle Disqus comments. Specify the
-                          Disqus sitename identifier here.
-``GITHUB_URL``            Your GitHub URL (if you have one). It will then
-                          use this information to create a GitHub ribbon.
-``GOOGLE_ANALYTICS``      Set to 'UA-XXXX-YYYY' to activate Google Analytics.
-``GOSQUARED_SITENAME``    Set to 'XXX-YYYYYY-X' to activate GoSquared.
-``MENUITEMS``             A list of tuples (Title, URL) for additional menu
-                          items to appear at the beginning of the main menu.
-``PIWIK_URL``             URL to your Piwik server - without 'http://' at the
-                          beginning.
-``PIWIK_SSL_URL``         If the SSL-URL differs from the normal Piwik-URL
-                          you have to include this setting too. (optional)
-``PIWIK_SITE_ID``         ID for the monitored website. You can find the ID
-                          in the Piwik admin interface > Settings > Websites.
-``LINKS``                 A list of tuples (Title, URL) for links to appear on
-                          the header.
-``SOCIAL``                A list of tuples (Title, URL) to appear in the
-                          "social" section.
-``TWITTER_USERNAME``      Allows for adding a button to articles to encourage
-                          others to tweet about them. Add your Twitter username
-                          if you want this button to appear.
-``LINKS_WIDGET_NAME``     Allows override of the name of the links widget.
-                          If not specified, defaults to "links".
-``SOCIAL_WIDGET_NAME``    Allows override of the name of the "social" widget.
-                          If not specified, defaults to "social".
-=======================   =======================================================
+=============================   =======================================================
+Setting name                    What does it do?
+=============================   =======================================================
+``SITESUBTITLE``                A subtitle to appear in the header.
+``DISQUS_SITENAME``             Pelican can handle Disqus comments. Specify the
+                                Disqus sitename identifier here.
+``GITHUB_URL``                  Your GitHub URL (if you have one). It will then
+                                use this information to create a GitHub ribbon.
+``GOOGLE_ANALYTICS``            Set to 'UA-XXXXXXXX-Y' to activate Google Analytics.
+``GOOGLE_ANALYTICS_PROPERTY``   Set to property like `auto` to fine-tune tracking code.
```

Comment:
Google refers to `UA-XXXXX-Y` as a "Property", so the name of this setting could be confusing. The [docs on this topic](https://developers.google.com/analytics/devguides/collection/analyticsjs/command-queue-reference#create) refer to this field as the "cookie domain" field, so perhaps we should rename this setting to `GA_COOKIE_DOMAIN`?